main/integration-tests/support/utils/generators.js

```diff
@@ -34,6 +34,7 @@ async function getGenerators({ setup }) {
     lastName: () => `TestUser${chance.last({ nationality: 'en' })}`,
     description: () => `Resource automatically created by SWB integration test - ${runId}`,
     defaultProjectId: () => setup.settings.get('projectId'),
+    defaultIndexId: async () => setup.getDefaultIndexId(),
```

Comment:
Hmm, this might be an issue. Because all generators are async, except this one.  What you can do is look up the value in the init method of setup and then use it, instead of looking it up every time we call a generator method.

Originally, I was thinking that defaultProjectId() does not belong to generator, since we are not actually generating this.  But, it was convenient making it available via the generator.

Reply:
Made it async.